packages/react-aria-components/stories/ColorPicker.stories.tsx

```diff
@@ -24,8 +24,10 @@ export default {
   title: 'React Aria Components'
 };
 
+type AllColorSpaces = ColorSpace | 'hex';
```

Comment:
This is only used once – does it merit adding?

Reply:
it's a story, not an official example or anything, so in general, not too fussed about this. I've removed it though because it was easy